components/carousel/CarouselTypeLatestMints.vue

```diff
@@ -14,7 +14,7 @@
           </nuxt-link>
 
           <div class="flex flex-col items-start">
-            <div class="flex justify-between items-center">
+            <div class="flex justify-between items-center hidden md:flex">
```

Comment:
isn't first `flex` necessary since it's hidden ?

Reply:
you are right, removed the duplicate  flex